src/products/views/ProductCreate/handlers.ts

```diff
@@ -85,7 +87,12 @@ export function createHandler(
   }) => Promise<FetchResult<ProductDeleteMutation>>,
 ) {
   return async (formData: ProductCreateData) => {
-    let errors: Array<AttributeErrorFragment | UploadErrorFragment> = [];
+    let errors: Array<Partial<
+      | AttributeErrorFragment
+      | UploadErrorFragment
+      | ProductErrorFragment
+      | ProductChannelListingErrorFragment
+    >> = [];
```

Comment:
I don't get what `Partial` is here for

Reply:
There is a `channels` field on `ProductChannelListingErrorFragment`

Reply:
Instead of handicapping returned types we can resolve problems where they actually appear. Try filtering errors in the components

Reply:
Ok, fixed